tracing/src/lib.rs

```diff
@@ -107,6 +107,8 @@
 //! tracing = "0.1"
```

Comment:
the rustdoc recommends 2 digits of semver where as the readme suggests 3, this should be made consistent as part of this PR I think.

Reply:
i think it should _probably_ be three digits as long as we remember to update it when increasing the version number

Reply:
(my current workflow for releasing new versions involves find-and-replace, so this _should_ happen...would be nice to have automation, of course)

Reply:
okay, do you want me to do apply that change everywhere in this PR?

Reply:
let's do that in a separate branch